wiki/introduction/usage.html

```diff
@@ -24,19 +24,19 @@ <h2 id="toc0">General Workflow</h2>
             <li>Post-processing (see <a href="/wiki/configuration/{{ site.wiki_version }}/categories">Categories</a> and <a href="/wiki/scripts/post-processing-scripts">User Scripts</a>)</li>
         </ul>
     </li>
-</ul>
+</ol>
 <h2 id="toc1">Adding an .nzb file to SABnzbd</h2>
 <ul>
     <li><strong>Upload an NZB</strong> - On the index of the web interface there's a box to upload an NZB you've downloaded to your hard drive. You can upload the raw .NZB file, or an NZB that has been zipped, gzipped or rar'ed.</li>
 </ul>
 <ul>
-    <li><strong>Open With SABnzbd</strong> - SABnzbd includes the ability to associate NZBs with SABnzbd. Simply have your browser open the NZB with SABnzbd, or set your OS to associate NZBs with SABnzbd.</li>
+    <li><strong>Open with SABnzbd</strong> - SABnzbd includes the ability to associate NZBs with SABnzbd. Simply have your browser open the NZB with SABnzbd, or set your OS to associate NZBs with SABnzbd.</li>
 </ul>
 <ul>
-    <li><strong>Add by NZB Dropbox</strong> - In Config -&gt; Folders, you can set a folder to be "Watched" by SABnzbd for downloaded NZBs. Once you've specified a directory to be watched, any NZBs (zipped, gzipped or rar'ed ones) will be automatically added to SABnzbd and deleted.</li>
```

Comment:
First thing I thought when I read "Dropbox" here, was the DropBox service. ;)

Reply:
Hahah, indeed. We do suggest that somewhere else I think, if you have a server seperate to use Dropbox on both computers to sync watched folders automatically. But not here.